src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -2,6 +2,7 @@
 
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.repackaged.com.google.gson.Gson;
```

Comment:
Look into what other library can be used for json serialization. repackaged.com.google seems a bit odd.

Reply:
Thanks! Fixed the import.